config/crd/patches/cainjection_in_ibmblockcsis.yaml

```diff
@@ -0,0 +1,7 @@
+# The following patch adds a directive for certmanager to inject CA into the CRD
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    cert-manager.io/inject-ca-from: $(CERTIFICATE_NAMESPACE)/$(CERTIFICATE_NAME)
+  name: ibmblockcsis.csi.ibm.com
```

Comment:
is this generated with a missing new line?

Reply:
is that a no?
so could you open a ticket and write its number here, to assert that all auto-generated files are exactly the same as the ones in the project?
it should be a blocker to this PR, since we should not maintain [different content](https://github.com/IBM/ibm-block-csi-operator/blob/5604e5bc0cf4cd7f3e887bf58170f328c8273b7c/config/crd/patches/labels_patch.yaml#L12) than the auto-generated one

Reply:
as per our conversation, specifically this file (and some others) are not relevant to said test, since they are generated only once

Reply:
removed this file and created the test that we talked about `hack/check-generated-manifests.sh`